src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/App.razor

```diff
@@ -0,0 +1,54 @@
+﻿@implements IRazorComponentApplication<App>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <title>Components-CSharp</title>
+    <base href="/" />
+    <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
+    <link href="css/app.css" rel="stylesheet" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <link href="Components-CSharp.styles.css" rel="stylesheet" />
+    <HeadOutlet />
+    @*#if PWA -->
+    <link href="manifest.webmanifest" rel="manifest" />
+    <link rel="apple-touch-icon" sizes="512x512" href="icon-512.png" />
+    <link rel="apple-touch-icon" sizes="192x192" href="icon-192.png" />
+    ##endif*@
+</head>
+
+<body>
+
+    <svg class="loading-progress">
+        <circle r="40%" cx="50%" cy="50%" />
+        <circle r="40%" cx="50%" cy="50%" />
+    </svg>
```

Comment:
Is the loading progress relevant if you're not using Blazor WebAssembly?

Reply:
Good point. This should probably not be there if you're not, even though it's then an extra (optional) thing for people to add if they choose to enable WebAssembly later.